OneCommunityGlobal/HGNRest
# yihan Create way to manually add in lost hours for past years of volunteers

# Description
<img width="751" alt="image" src="https://github.com/OneCommunityGlobal/HGNRest/assets/94303659/83165a8a-81f0-4b07-aa1a-c5f6fdf4a056">

## Related PRS (if any):
To test this backend PR you need to checkout the[ #1535](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1535) frontend PR.

## Main changes explained:
- Update timeentry.js for including new properties.
- Update timeentryRouter.js and timeEntryController.js for updating exist functions and adding new functions that related to time entries.
- Update dashboardhelper.js, taskHelper.js and userHelper.js for filtering unrelated timeEntries.

## How to test:
1. check into current branch on both FE and BE.
2. do `npm run build` and `npm start` to run this PR locally
3. follow test steps in [#1535](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1535).
